.github/workflows/standard-its.yml

```diff
@@ -106,9 +106,7 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       MVN: mvn --no-snapshot-updates
-      # Added -Dcyclonedx.skip=true to avoid ISO-8859-1 [ERROR]s
-      # May be fixed in the future
-      MAVEN_SKIP: -P skip-static-checks -Dweb.console.skip=true -Dmaven.javadoc.skip=true -Dcyclonedx.skip=true
+      MAVEN_SKIP: -P skip-static-checks -Dweb.console.skip=true -Dmaven.javadoc.skip=true
```

Comment:
Comments are out of order. Can also skip CycloneDX for the standard ITs since we do it for the build. It doesn't add much to repeat it for the ITs as well.